crates/rpc/rpc-types/src/mev.rs

```diff
@@ -613,9 +613,9 @@ pub struct EthCallBundle {
 #[derive(Default, Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct EthCallBundleResponse {
+    pub bundle_hash: String,
```

Comment:
any reason why this shouldn't be a H256?

Reply:
I believe there's no spec on how this computed, but I believe it's always B256